packages/react/src/hooks/useAmplify.ts

```diff
@@ -3,10 +3,10 @@ import * as React from 'react';
 import { Theme } from '@aws-amplify/ui';
 
 import { AmplifyContext } from '../components/AmplifyProvider/AmplifyContext';
-import * as primitives from '../primitives';
+import * as primitives from '../primitives/primitives';
```

Comment:
Feels awkward to export from `primitives/primitives`

Reply:
Rename to `primitives/components.ts`

Reply:
This file name feels better